packages/backend/src/server/web/feed.ts

```diff
@@ -39,20 +41,61 @@ export default async function(user: User) {
 	});
 
 	for (const note of notes) {
-		const files = note.fileIds.length > 0 ? await DriveFiles.findBy({
-			id: In(note.fileIds),
-		}) : [];
-		const file = files.find(file => file.type.startsWith('image/'));
+		let contentStr = await noteToString(note, true);
+		let next = note.renoteId ? note.renoteId : note.replyId;
+		let depth = history;
+		while (depth > 0 && next && withAll) {
+			const finding = await findById(next);
+			contentStr += finding.text;
+			next = finding.next;
+			depth -= 1;
+		}
 
 		feed.addItem({
-			title: `New note by ${author.name}`,
+			title: `${author.name} ${(note.renoteId ? 'renotes' : (note.replyId ? 'replies' : 'says'))}: ${note.cw ? note.cw : (note.text ? note.text : 'post a new note')}`.replace(/[\x00-\x08\x0b\x0c\x0e-\x1f\x7f]/g, '').substring(0,100),
 			link: `${config.url}/notes/${note.id}`,
 			date: note.createdAt,
-			description: note.cw || undefined,
-			content: note.text || undefined,
-			image: file ? DriveFiles.getPublicUrl(file) || undefined : undefined,
+			description: note.cw ? note.cw.replace(/[\x00-\x08\x0b\x0c\x0e-\x1f\x7f]/g, '') : undefined,
+			content: `${contentStr.replace(/[\x00-\x08\x0b\x0c\x0e-\x1f\x7f]/g, '')}`
```

Comment:
There is no need to use a template string here.
```suggestion
			content: contentStr.replace(/[\x00-\x08\x0b\x0c\x0e-\x1f\x7f]/g, '')
```